Extensions/Effects/crt-pixi-filter.js

```diff
@@ -5,8 +5,8 @@ gdjs.PixiFiltersTools.registerFilterCreator('CRT', {
     return crtFilter;
   },
   update: function(filter, layer) {
-    if (filter.animated) {
-      filter.time += layer.getElapsedTime() / 1000;
+    if (filter.animationSpeed !== 0) { 
+      filter.time += layer.getElapsedTime() / 1000 * filter.animationSpeed;
       filter.seed = Math.random();
```

Comment:
the animation speed multiplier approach doesnt seem to make a difference for this effect. I wonder if we need limit the frequency at which seed is being changed to to it

Reply:
Good catch! seed and time must be handled separately I think. One with frequency, the other with the animation speed.
So this means two settings in the UI :) 

If that does not make sense to have them separated, then we must update the seed every once in a while, but that means choosing a frequency for the user so seems weird.... It would be great to know what is the difference between time and seed for this CRT filter :)

Reply:
It works but not obvious. It's much more obvious when `lineWidth` is 10.

The animation of `crt` consists of `noise`(influenced by `seed`) and `line moves`(influenced by `time`).

Reply:
I came up with this formula
filter.seed =  Math.round((filter.time % filter.animationSpeed) * 10) + 0.1;

seems to do the trick :)

Reply:
> If we can give the user control over the speed without adding complexity, that would be great
I do agree with it. But this is a problem with this formula(`oldFilmFilter` as an example), scratches used to be appeared randomly, but now it always appear in the same position.

Reply:
> > If we can give the user control over the speed without adding complexity, that would be great
> 
> I do agree with it. But this is a problem with this formula(`oldFilmFilter` as an example), scratches used to be appeared randomly, but now it always appears in the same position. So I think `random` cannot be removed. What do you think?

I think the frequency is kind of counterintuitive, since a greater value results in slower animation. I think we can improve it to make it consistent :)

Reply:
put this in old film filter- exact same logic, but with your  renamed parameters
```
update: function(filter, layer) {
    if (filter.animationFrequency !== 0) {
      filter._animationTimer +=
        (layer.getElapsedTime() / 1000) * filter.animationFrequency;
      filter.seed =
        Math.round((filter._animationTimer % filter.animationFrequency) * 10) +
        0.1;
    }
  },
```
the formula works and a greater value results in faster animation, not the other way around

Reply:
> Math.round((filter._animationTimer % filter.animationFrequency) * 10) + 0.1;

What are the reason for choosing 10 and 0.1?
This seems to also severly limit the 

> filter._animationTimer += (layer.getElapsedTime() / 1000) * filter.animationFrequency;

A frequency is not the same unit as a multiplier, which does not have a unit 😬
A frequency unit is "per second". We should at least be following proper units or not call this a frequency.

Not saying that it's not a good idea to have the two tied - but here we're getting result that is not predictable for the end user.

> the formula works and a greater value results in faster animation, not the other way around

This is the same with a frequency and an animation speed right?
2x, 3x, 4x => you're increasing the speed of the animation

20hz, 30hz, 60hz => you're increasing the speed of the update to the random noise shown on screen.

Reply:
> The animation of crt consists of noise(influenced by seed) and line moves(influenced by time).

This seems that the CRT filter has two things: noise (seed) and how line moves (time).
If they are two different things, why not showing the two of them in the UI?

Noise update: 30hz, 60hz, 120hz...
Line movement speed: 0.5x, 1x, 2x...

Sounds easy no? And very flexible :) We're not limiting the effect compared to the original Pixi effect, and we're using units that are the proper one.

Reply:
I guess it would be even better to keep the randomness, but also keep it animationSpeed somehow?

Reply:
Yep! That's why I think we need both, but separate (only in the case of CRT I think):

```
update: function(filter, layer) {
    // Update time (for lines on screen)
    filter.time += layer.getElapsedTime() / 1000 * filter.animationSpeed;

    // Update random scratches (at the specified frequency)
    if (filter.animationFrequency !== 0) { 
      filter._animationTimer += layer.getElapsedTime() / 1000;
      if (filter._animationTimer >= 1 / filter.animationFrequency) {
        filter.seed = Math.random();
        filter._animationTimer = 0;
      }
  }
},
```

Reply:
Thank you! And I have fixed it.

Reply:
I like the idea of splitting the random seed from time and letting the user set them individually. If this approach also means that for both a higher value results in a faster animation, then I'm good with it :) one thing worth double checking is if the default values that are set when adding each filter best demonstrate it's animation

Reply:
Also check if changing each results in a visually different result. I know it's kind of obvious, but worth double checking

Reply:
> Also check if changing each results in a visually different result. I know it's kind of obvious, but worth double checking

I have checked them and no problems.

>if the default values that are set when adding each filter best demonstrate it's animation

The default value of `lineWidth`, `lineContrast`, `noiseSize` in `CRT` may need to change. But I don't know which value is appropriate because it looks different in different game. 
I have tried it and I think `lineWidth: 3, lineContrast: 0.2, noiseSize: 2` looks fine to me. What do you think?

Reply:
> I have tried it and I think lineWidth: 3, lineContrast: 0.2, noiseSize: 2 looks fine to me

Add a screenshot so that we can all see the result :) I would say that any default value that make a pretty rendering on screen are fine :)

Reply:
There are three examples.
### Pixi-filter documentation
![Screenshot from 2020-03-22 20-40-45](https://user-images.githubusercontent.com/32167255/77249619-7c479800-6c7d-11ea-951e-1bd031c1d529.png)

### Default value we used to use
![Screenshot from 2020-03-22 20-27-21](https://user-images.githubusercontent.com/32167255/77249670-c7fa4180-6c7d-11ea-95df-f12326e65e9d.png)

### Pixi-filter Demo
![Screenshot from 2020-03-22 20-40-45](https://user-images.githubusercontent.com/32167255/77249922-78b51080-6c7f-11ea-80af-62ee347aaa3e.png)

It is hard to say which looks best to me. Can you choose one? And I will both fix it and the default frequency.

Reply:
I'm fine with either pixi-filter doc/demo :)